rmf_traffic/src/rmf_traffic/schedule/RectifierInternal.hpp

```diff
@@ -19,7 +19,7 @@
 #define SRC__RMF_TRAFFIC__SCHEDULE__RECTIFIERINTERNAL_HPP
```

Comment:
Should we rename this file to `internal_Rectifier.hpp` for consistency?

Reply:
Good idea, updated: e3e9c05